indexer-app/src/cmr/indexer/system.clj

```diff
@@ -46,6 +46,11 @@
   {:default 5
    :type Long})
 
+(def index-set-mappings-cubby-keys
+  "The list of keys related to index-set-mappings which should be deleted when we want to clear
+  the index-set-mappings cache."
+  [":concept-mapping-types-hash-code" ":concept-indices-hash-code"])
```

Comment:
Sidenote: yikes ... we use stringifyed keywords in CMR?

Reply:
Yeah... I thought about cleaning that up, but since it is just cosmetic I didn't want to think through transition implications if we use two different keys for the same cached item while both the old and new version of apps are running.